.github/workflows/macos-build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-13, macos-14]
+        os: [macos-14]
```

Comment:
This needs to be reverted. macos-13 is x86 and macos-14 is arm. They should be both tested until GitHub drops x86 support.